app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,9 +1,9 @@
-class SessionsController < ApplicationController
+class Api::V1::SessionsController < ApplicationController
```

Comment:
necessary change; need to remember that in the future when creating non-default controllers (not using rails g)